### PR TITLE
Add app open ai PROOF-6

### PR DESCRIPTION
**Pull request by SgtMarmite**
I have added app-open-ai docs. I have also created new category in the navbar "applications/other", since app-open-ai is not a trigger.

### CODE REVIEW

**Comment by SgtMarmite** [comment]
tagging @bakobako

File: components/applications/ai/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: AI Applications
+permalink: /components/applications/ai/
+redirect_from:
+    - /applications/ai/
+
+--- 
+
```
Comment (bakobako) [inline components/applications/ai/index.md:8]: I would add a short description of this section.
e.g.

AI applications are used to generate, process, or enrich data by custom models or 3rd Party AI API services. The following AI applications are available:


I would also add a very short description of what the OpenAI app does 
e.g.

- [OpenAI](/components/applications/ai/open-ai/) - Process input data with a custom prompt with OpenAI models

**Review by bakobako — CHANGES_REQUESTED** [review]

File: components/applications/ai/index.md (hunk 1 above)
Comment (bakobako) [reply components/applications/ai/index.md:8]: @SgtMarmite tagging you

**Comment by SgtMarmite** [comment]
@bakobako Added in https://github.com/keboola/connection-docs/pull/464/commits/e28c05237b07e3178b2746e23d90b6d5dda06080

**Review by hhanova — APPROVED** [review]
suggested UI changes :-) 
(Please create new screenshots when done. Thanks a lot!)

Screenshot #1
1. Provide a table containing columns with input data to be used in **a** prompt and other arbitrary columns which can be used as **a** primary key of the output table.
2. The application will query **the** Open AI service and store the result in the ABC column for each input.
3. extractor configuration => **Data Source Configuration**
4. Hide **data source** configuration settings
5. Sleep **T**ime

Screenshot #2
Pick from predefined models or enter a custom one. => **Select which type of model you want to use, either predefined or custom.**

Screenshot #3
1. Refer to **the** input column using placeholder xyz.
2. Za poslední větou chybí tečka.
3. You can enter multiple columns at once, e.g., ....

**Comment by SgtMarmite** [comment]
@hhanova I have edited the UI and made new screenshots

**Review by bakobako — APPROVED** [review]

**Pull request merged**